images-list

```diff
@@ -120,20 +153,29 @@ quay.io/cilium/operator-aws rancher/mirrored-cilium-operator-aws v1.9.4
 quay.io/cilium/operator-azure rancher/mirrored-cilium-operator-azure v1.9.4
 quay.io/cilium/operator-generic rancher/mirrored-cilium-operator-generic v1.9.4
 quay.io/cilium/startup-script rancher/mirrored-cilium-startup-script 62bfbe88c17778aad7bef9fa57ff9e2d4a9ba0d8
+quay.io/coreos/kube-state-metrics rancher/mirrored-coreos-kube-state-metrics v1.9.7
 quay.io/coreos/prometheus-config-reloader rancher/coreos-prometheus-config-reloader v0.39.0
+quay.io/coreos/prometheus-config-reloader rancher/mirrored-coreos-prometheus-config-reloader v0.38.1
 quay.io/coreos/prometheus-operator rancher/coreos-prometheus-operator v0.39.0
+quay.io/coreos/prometheus-operator rancher/mirrored-coreos-prometheus-operator v0.38.1
 quay.io/kiali/kiali rancher/kiali-kiali v1.22.1
 quay.io/kiali/kiali rancher/kiali-kiali v1.23.0
 quay.io/kiali/kiali rancher/kiali-kiali v1.24.0
 quay.io/kiali/kiali rancher/kiali-kiali v1.28.0
 quay.io/kiali/kiali rancher/kiali-kiali v1.29.0
+quay.io/kiali/kiali rancher/mirrored-kiali-kiali v1.29.0
 quay.io/prometheus-operator/prometheus-config-reloader rancher/coreos-prometheus-config-reloader v0.43.0
+quay.io/prometheus-operator/prometheus-config-reloader rancher/mirrored-coreos-prometheus-config-reloader v0.43.0
```

Comment:
Did we find out why its coming from 2 different origins ? Technically we shouldn't be doing this. Was it for ease of chart change ?

Reply:
No, I asked Arvind about something else, but forgot to ask about this.

Reply:
@aiyengar2 you remember why we did this ?

Reply:
There were three that I found that were like this. I removed the ones that were not explicitly mentioned in https://github.com/rancher/charts/pull/1003. Those were:
prom/prometheus rancher/mirrored-prom-prometheus v2.18.2
quay.io/prometheus-operator/prometheus-operator rancher/mirrored-coreos-prometheus-operator v0.43.0
quay.io/prometheus-operator/prometheus-config-reloader rancher/mirrored-coreos-prometheus-config-reloader v0.43.0

I can re-add them if necessary.

Reply:
The first one is used in system-charts, that will be updated later. The other 2 come from this PR: https://github.com/rancher/image-mirror/pull/47/files
Looks like that isn't in yet, so I don't think we need to mirror those. When we do we should try to put them in the right org.

Reply:
Confirmed that we don't need those yet. 0.39.x is used in system-charts right now but we aren't mirroring those for 2.5.6. And the 0.43 ones aren't migrated to yet.